src/main/kotlin/com/shortener/domain/UrlEntry.kt

```diff
@@ -0,0 +1,61 @@
+package com.shortener.domain
+
+import org.springframework.format.annotation.DateTimeFormat
+import java.time.LocalDateTime
+import javax.persistence.*
+import javax.validation.constraints.NotBlank
+import javax.validation.constraints.NotNull
+import javax.validation.constraints.Size
+
+@Entity(name = "URL_ENTRIES")
+class UrlEntry {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    @Column(name = "id")
+    private var id: Long? = null
+
+    @Column(name = "long_url")
+    @Size(max = 2048)
+    @NotBlank
+    private var longUrl: String? = null
+
+    @Column(name = "created_at")
+    @DateTimeFormat(iso = DateTimeFormat.ISO.DATE_TIME)
+    @NotNull
+    private var createdAt: LocalDateTime? = null
```

Comment:
making all of them null by default is really a bad idea imho. (you even use the notNull annotation). Why not use a proper value? 

P.S: look into the billion dollar mistake, the way you use it is for simplicity, and not because null is an actuall accepted value.
https://en.wikipedia.org/wiki/Tony_Hoare

Reply:
ok, and what should be the proper value for id, for example ?